SGoopas/Assets/Scripts/Game/3D/ObjectInterract/ObjInteractable.cs

```diff
@@ -62,11 +62,8 @@ public override void InteractionEnded()
         switch (objType)
         {
             case ObjectType.pushPull:
```

Comment:
you can delete this whole method and any references to it

Reply:
this function is tied with many other functions including those in switch interactables. I tried deleting them but it makes it so that no object can be interacted with. I believe we should just leave it as it is

Reply:
This method only returns true now.. so it’s useless

Reply:
yea thats what I thought too. But I think when I removed it, and all the reference to it, something got broken somewhere. The code compiles, but the logic condition that oversees it gets broken. I currently do not have enough time to go through each line to find the logic fault unfortunately